client/host.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"fmt"
+	"os"
+)
+
+type Host struct {
+	Id        string `json:"id"`
+	NameLabel string `json:"name_label"`
+}
+
+func (h Host) Compare(obj interface{}) bool {
+	otherHost := obj.(Host)
+	if otherHost.NameLabel != "" && h.NameLabel == otherHost.NameLabel {
+		return true
+	}
+	return false
+}
+
+func (c *Client) GetHostByName(nameLabel string) (hosts []Host, err error) {
+	obj, err := c.FindFromGetAllObjects(Host{NameLabel: nameLabel})
+	if err != nil {
+		return
+	}
+	return obj.([]Host), nil
+}
+
+func FindHostForTests(host *Host) {
+	hostName, found := os.LookupEnv("XOA_HOST")
```

Comment:
I would like to be careful about the amount of "inputs" the acceptance test suite requires. In this case, whoever is running the test suite will need to provide a particular host. This is more brittle than passing in a particular template or pool.

I think it would be better to use `accTestPool.Master` to look up the host of the primary instance in the pool and use that host in the tests instead.